docker-compose.yml

```diff
@@ -142,11 +122,6 @@ services:
       context: .
```

Comment:
Actually, the `home` service isn't used on production, so let's move this entire service to `override`